### ISSUE TITLE
new console warning in 2.2.0

### ISSUE THREAD

**Issue by frehner** [optimizely/react-sdk#60]
I'm getting a new warning in 2.2.0 that doesn't appear if I downgrade to 2.1.0. It's not quite clear what the issue is based on the warning so I'm not sure how to fix it:

`[OPTIMIZELY] - WARN  2020-08-05T16:28:18.546Z BUCKETER: Bucketed into an invalid variation ID. Returning null.`

My implementation looks like the following:

```jsx
const optimizely = createInstance({
  sdkKey: key,
  logLevel: 3,
})

function App() {
  const user = useUser()
  return <OptimizelyProvider
      optimizely={optimizely}
      timeout={500}
      user={{ id: user ? user.email : '' }} //have to wait for the user api to resolve
  >
    // rest of app as children
  </OptimizelyProvider>
}
```

also, we're only using the feature toggles at the moment in our app, nothing else. 

Can you guide me as to what the warning is and how to correct it? Thanks.

**Comment by Danny-Driscoll** [comment]
Thanks for raising, this warning-level log is being raised in error on Experiments and Feature Rollouts at 0% Traffic allocation at the moment.  Based on our investigation it is a logging-specific issue and does not reflect any configured Features or Experiments being delivered incorrectly. We are planning to release a fix soon and will update here once complete.

**Comment by frehner** [comment]
wondering if there's been any progress in this area? thanks! 🙂

**Comment by zashraf1985** [comment]
This has been fixed in version [2.3.0](https://github.com/optimizely/react-sdk/releases/tag/2.3.0)
